Makefile

```diff
@@ -31,3 +32,16 @@ tag:
 .PHONY: docker-build
 docker-build: test ## Build docker image with the manager.
 	docker build -t ${IMG} .
+
+bin:
+	mkdir $(BINDIR)
+
+.PHONY: download-spdx-sbom-generator
+download-spdx-sbom-generator: bin
+	curl -L -o $(BINDIR)/spdx-sbom-generator-v0.0.15-linux-amd64.tar.gz https://github.com/opensbom-generator/spdx-sbom-generator/releases/download/v0.0.15/spdx-sbom-generator-v0.0.15-linux-amd64.tar.gz
+	tar -xf ./$(BINDIR)/spdx-sbom-generator-v0.0.15-linux-amd64.tar.gz --directory $(BINDIR)
+
+
+.PHONY: sbom-generator
```

Comment:
nitpicking, I think this line is not needed/wrong

Reply:
Fixed.